src/main/java/picocli/CommandLine.java

```diff
@@ -5864,6 +5903,8 @@ public void clear() {
             private Integer exitCodeOnInvalidInput;
             private Integer exitCodeOnExecutionException;
 
+            private IModelTransformer modelTransformer = null;
```

Comment:
Can we add public getter and setter accessors for this attribute to `CommandSpec`?
Following the naming convention I use here, that would be

```java
/** Some javadoc here
  * @since 4.6 */
public IModelTransformer modelTransformer() { return modelTransformer; }

/** Some javadoc here
  * @since 4.6 */
public CommandSpec modelTransformer(IModelTransformer modelTransformer) { this.modelTransformer = modelTransformer; return this; }

Reply:
That's ok, than we need to add public API to CommandLine to manually trigger the transformation ? I'll see into this.

Reply:
Actually you don't need programmatic API for this :) because you're constructing command line on the fly ( i.e. your code does construction ) , but maybe it can serve some purpose.

Reply:
I've done it.